projects.tex

```diff
@@ -1,36 +1,32 @@
-\section{PROJECTS}
+\section{MENTORSHIP \& PROJECTS}
 \begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
-  Fluoroscopy Simulator – Medical College of Wisconsin & September 2019 – May 2020
+  FIRST Mentor – Charger Robotics Team 537 & October 2021 – Present
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Created UI mockups detailing the user workflow and consulted with UX professors and application users to
-        iterate on the design prior to starting development work on the application.
+        Taught programming concepts and problem solving skills to students, allowing them to continue learning independently
+        and to contribute to the robot's codebase.
     }
     \item{
-        Architected the full-stack web application solution based on requirement gathering sessions with the product
-        owner and other stakeholders, improving on the original POC developed prior to our team’s involvement.
+        Worked with students and department leaders to develop leadership qualities, enabling them to teach each other and
```

Comment:
Collaborated or "liaised between" is a little better than "worked" but you could also stand to rewrite this to be a little more impactful, e.g:
- Developed students into future leaders, teaching them to knowledge-share, build each others' skills and collaborate.
- 
I think you could also mention your role as a conduit between teacher and other advisors with something like -

- Liaised between senior mentors, department leaders and students to foster an environment of open communication, learning and community, resulting in a well-informed, close-knit team and high morale.

Reply:
I love that second bullet! Added in c779aa8fd047153d7179eedc881201d497159268